portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,10 +31,22 @@
 /** Servlet that returns some example content and handles data. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private List<String> commentsStorage = new ArrayList<>();
+  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  int max_comments_num = 3;
```

Comment:
What would happen if  multiple people connect to the service?

Ideally servlets should be stateless.

Reply:
Yes, this comment makes sense! I also consider to store max_comments_num for each viewer (without setting the global variable), while still thinking how to get the storage of a specific user ID. I will come back to the solution soon.

Reply:
With advice from the technical support, max_comments_num has now been stored as a property in datastore for each viewer as well. The maximum number among all these stored number will be treated as the bound of shown comments' number. I will continue working on trying to make these codes clean.